qiime2/sdk/plugin_manager.py

```diff
@@ -52,6 +52,7 @@ def _init(self):
         self.formats = {}
         self.views = {}
         self.type_formats = []
+        self.importable_types = set()
```

Comment:
Please rename and change the data type:

```suggestion
        self.types = {}
```

:mount_fuji: